dallinger/experiment_server/experiment_server.py

```diff
@@ -61,6 +60,9 @@
 
 @app.before_first_request
 def _config():
+    app.config["SECRET_KEY"] = os.environ.get(
+        "FLASK_SECRET_KEY", "THIS IS A BAD SECRET"
```

Comment:
Why the default? (If for some reason the key was missing, wouldn't we want to fail?)

Reply:
Tests, though I guess there's an easy workaround for that.